docs/clipboad.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Get/Set Clipboard
+---
+
+## Get Clipboard
+
+For iOS 13+ real devices, Apple security preferences require the WebDriverAgentRunner application to be in foreground in order to be able to receive the system clipboard content.
+Otherwise an empty string is always returned.
```

Comment:
actually, in recent iOS releases it may throw an error instead

Reply:
Hm, iOS 16.5 and iPadOS 16.6 beta returned an empty string in my local case, so not sure when it raise an exception

Reply:
Yea, I wondered if it was something timing stuff..., or under a specific condition..